nixos/tests/installer.nix

```diff
@@ -51,7 +51,7 @@ let
           boot.loader.systemd-boot.enable = true;
         ''}
 
-        boot.initrd.secrets."/etc/secret" = /etc/nixos/secret;
+        boot.initrd.secrets."/etc/secret" = ./secret;
```

Comment:
You can revert this line to fix the test for systemd-boot, for now.

Reply:
OK, done in 8d45d82c71b